python_modules/dagster/dagster/_utils/__init__.py

```diff
@@ -421,9 +421,10 @@ def touch_file(path):
         os.utime(path, None)
 
 
-def _kill_on_event(termination_event):
+def _kill_on_event(termination_event, done_event: threading.Event):
```

Comment:
this is like `terminate_if_not_done_event` now right?

Reply:
yea took a naming pass in response